renderer/brave_content_settings_agent_impl_browsertest.cc

```diff
@@ -427,17 +484,78 @@ IN_PROC_BROWSER_TEST_F(BraveContentSettingsAgentImplBrowserTest,
   EXPECT_EQ(ExecScriptGetStr(kReferrerScript, contents()), "");
   EXPECT_TRUE(GetLastReferrer(url()).empty());
 
-  // Sub-resources loaded within the page get their referrer spoofed.
-  EXPECT_EQ(ExecScriptGetStr(create_image_script(), contents()),
-            image_url().spec());
-  EXPECT_EQ(GetLastReferrer(image_url()), iframe_url().GetOrigin().spec());
+  // Same-site sub-resources within the page get the page origin as referrer.
```

Comment:
@fmarier There is some confusion, since the the spec defines "same-site" as equal domains. This way, navigations to subdomains are not same-site, so we probably should rewrite these comments.

Reply:
Do you mean [same-origin](https://www.w3.org/TR/referrer-policy/#referrer-policy-same-origin)?

Because I'm using "same-site" here as defined in the [Fetch spec](https://html.spec.whatwg.org/multipage/origin.html#schemelessly-same-site) since our policy is specifically looser (for webcompat reasons).

Reply:
yup, sorry, you are correct!